packages/react-core/src/helpers/util.ts

```diff
@@ -266,3 +266,9 @@ export const toCamel = (s: string) => s.replace(/([-_][a-z])/gi, camelize);
  * Copied from exenv
  */
 export const canUseDOM = !!(typeof window !== 'undefined' && window.document && window.document.createElement);
+
+/**
+ *
+ * @param {string} s string to make first letter a capital
+ */
+export const capitalizeFirstLetter = (s: string) => s.charAt(0).toUpperCase() + s.slice(1);
```

Comment:
There's a `capitalize` helper on line 9 already that does the exact same thing.

Reply:
I see that now.  I just read the description and thought it capitalized the whole string.  Thanks!  I will update description of original function.